src/framework/midi/midievent.h

```diff
@@ -767,11 +767,13 @@ struct Event {
                 break;
             //D3.3
             case Opcode::ControlChange: {
-                std::set<uint8_t> skip = { 6, 38, 98, 99, 100, 101 };
-                if (skip.find(index()) == skip.end()) {
-                    break;
-                }
                 switch (index()) {
+                default:
```

Comment:
Put `default` case to the beginning for code readability, as this is the standard case and not a "last resort fallback". The other cases are specializations.